src/System.Net.Requests/tests/HttpRequestResponseSerializationTest.cs

```diff
@@ -0,0 +1,34 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.Serialization.Formatters.Tests;
+using Xunit;
+
+namespace System.Net.HttpRequest.Tests
+{
+    public partial class HttpRequestResponseSerialization
+    {
+        [OuterLoop]
+        [Theory]
+        public void SerializeDeserialize_Roundtrips(Uri remoteServer)
+        {
+            HttpWebRequest request = WebRequest.CreateHttp(remoteServer);
+            request.KeepAlive = true;
+            request.CookieContainer = new CookieContainer(10);
+            HttpWebRequest clone = BinaryFormatterHelpers.Clone(request);
+            Assert.NotSame(request, clone);
+            Assert.Equal(request, clone);
```

Comment:
I see , . the timeout is in HttpClient. Now it's straight forward , i will  add it and update the PR